flake.nix

```diff
@@ -0,0 +1,75 @@
+{
+  description = "Seabug frontend";
+  inputs = {
+    nixpkgs.url = github:NixOs/nixpkgs/nixos-unstable;
+    dream2nix = {
+      url = github:nix-community/dream2nix;
+      inputs.nixpgks.follows = "nixpkgs";
```

Comment:
```suggestion
      inputs.nixpkgs.follows = "nixpkgs";
```

Reply:
Ooops, anyway weird that it did not complain to follow a non existing input.